model/project_ref.go

```diff
@@ -1895,16 +1885,43 @@ func (projectRef *ProjectRef) CanEnableCommitQueue() (bool, error) {
 }
 
 // Upsert updates the project ref in the db if an entry already exists,
-// overwriting the existing ref. If no project ref exists, one is created
-func (projectRef *ProjectRef) Upsert() error {
-	_, err := db.Upsert(
-		ProjectRefCollection,
-		bson.M{
-			ProjectRefIdKey: projectRef.Id,
-		}, projectRef)
+// overwriting the existing ref. If no project ref exists, a new one is created.
+func (p *ProjectRef) Upsert() error {
+	if p.Private == nil {
+		// Projects are private by default unless they've been specially made
+		// public.
+		p.Private = utility.TruePtr()
```

Comment:
Good catch -- we can also change https://github.com/evergreen-ci/evergreen/blob/11dd59ac7f6f7b824d0cf99c7590692dae1d01d9/model/project_ref.go#L348 to use FromBoolTPtr

Reply:
Done.